index.ts

```diff
@@ -220,12 +226,14 @@ export class CapabilitiesController extends BaseController<any, any> implements
       const segments = method.split('_');
       let managed = '';
 
-      while (segments.length > 0 && !managedMethods.includes(managed)) {
+      while (segments.length > 0 && !managedMethods.includes(managed) && !wildCardMethodsWithoutWildCard[managed]) {
         managed += segments.shift() + '_';
       }
 
       if (managedMethods.includes(managed)) {
         return managed;
+      } else if (wildCardMethodsWithoutWildCard[managed]) {
+        return managed + '*';
       } else {
         return '';
       }
```

Comment:
This was not introduced by this PR, but the if/else statement beginning with `if (method.indexOf('_') > 0)` is pointless. Its else clause is simply `return managedMethods.includes(method) ? method : '';`, but this is already handled by the if statement [here, on R217](https://github.com/MetaMask/json-rpc-capabilities-middleware/pull/56/files#diff-ed009b6b86b017532ef0489c77de5100R217).

The else clause should be removed, and we could instead replace the if statement on R224 with:
```js
if (method.indexOf('_') === -1) {
  return ''
}
```

Reply:
fixed in 064c83b